### ISSUE TITLE
Master List of February 2017 Updates

### ISSUE THREAD

**Issue by edrushka** [mozilla/mozillafestival.org#488]
## Top Nav Bar  
https://mozillafestival.org/ (all pages)

- [x] Update wordmark design #499
- [x] Replace wordmark with updated version #500
- [x] Remove Tabzilla #501
- [x] Make blog first (furthest left) item in top nav bar

## Landing Page
Ticket #503
- [x] Update main banner
- [x] Update Speaker Series section
- [x] Design new section for blogs/news #502
- [x] Implement new section for blogs/news #505
- [x] Update grey CTA banner

## What to Expect Page
Ticket: #489
- [x] Update 'What is MozFest like?'
- [x] Replace video with 2017 version
- [x] Update Weekend Outline
- [x] Fix link in Participating at MozFest

## Spaces & Sessions Page
Ticket #490
- [x]  Update Spaces title to "MozFest 2016 Spaces"

## Tweet MozFest
Ticket #491
- [x] Update autofilled tweet text 

## Contact
Ticket #492
- [x] Update 'Sign up here' to link to 2017 tito page

## Volunteer
Ticket #493
- [x] Update text

## Fringe
Ticket #494
- [x] Update copy 
- [x] Update CTA button text and link
- [x] Remove (hide) events and form from page

## About
Ticket #495
- [x] Update copy in stats section
- [x] Update timeline

## Festival Team
Ticket #496
- [x] Remove entries for Marc and Spike 
- [x] Update text for Kevin, Sarah and Erika 

## Terms
Ticket #497
- [x] Update to https://www.mozilla.org/en-US/about/legal/terms/mozilla/

## Footer
- [x] Remove 'Read the blog' from footer #498
- [x] Implement add'l footer changes #504

**Comment by hannahkane** [comment]
@edrushka and @alanmoo - wondering if we can do a kind of virtual stand-up today in this ticket. I'm specifically curious about:
 * the status of the video, which affects #489 and I *think* #503 (because the single "watch video" button should point to the new video, right?)
 * the status of [selecting a new photo for the landing page](https://github.com/mozilla/mozillafestival.org/issues/508) which affects #503 
 * @alanmoo - how you're feeling about the remaining tickets, given the refactoring you did this week

**Comment by hannahkane** [comment]
My stand-up report is:
 * filed PRs for everything I think I can do on my own: parts of the landing page, everything on the What to Expect page except for the video, Spaces and Sessions page, Contact, Volunteer, Fringe, About, Team, and Terms

**Comment by alanmoo** [comment]
I refactored a fair amount of the site to use the newest versions of various libraries (react, react-router, mofo-bootstrap, and brought in mofo-ui). This allowed me to drop in the new footer fairly easily, and will allow other changes more easily going forward. I have a PR almost ready to add said footer and remove tabzilla.

I got hung up a little bit on the fringe events; something about recent changes broke how they interact with google sheets. But since we're disabling that functionality for now anyway, I'm going to come back to it when we actually know we need to use sheets again.

**Comment by edrushka** [comment]
The video is ready! https://vimeo.com/205552025/37560e3619 I've updated #489 and #503 

We've narrowed the main banner image down to 2 choices and have asked Sabrina to mock them up so we can make our final decision. See progress in #508

**Comment by hannahkane** [comment]
Woohoo! Seems like we're on track to push all of this to production on March 1. Let's check in again on Monday to see what remains to do.

**Comment by edrushka** [comment]
Actually, if possible, can we plan to push everything to prod (except #491) on Tuesday? We'd like to start tweeting on Wed and it would be great to have the page up and running for a bit before we push traffic there.

**Comment by edrushka** [comment]
Hero image for main banner is ready in #503

**Comment by hannahkane** [comment]
@alanmoo - what do you think about Erika's suggested timeline ^?

**Comment by alanmoo** [comment]
Might be a little bit of a push but I'll see if I can get it going today. I've been checking items off the main list as I open PRs for them. Might need some dev help reviewing them.

**Comment by hannahkane** [comment]
I updated the checklist just now with PRs I've filed. 

@mmmavis - are you available to do code review on Mozfest site stuff today, or should I try to track someone else down?

**Comment by mmmavis** [comment]
@hannahkane yup yup! what are the PRs I should review? I marked 3 as approved last week - they are just not merged yet.

**Comment by hannahkane** [comment]
Actually, seems like you've done all that can be done for now (thanks!), but more likely coming today, with a goal of pushing to production tomorrow. Thank you!

**Comment by mmmavis** [comment]
Cool just assign me as PR reviewer!

**Comment by alanmoo** [comment]
@edrushka Just as an update: I'm waiting on approval for #511 and #505 from both design and code standpoints. (#505 might need some minor graphic updates per branding guidelines). Once those two are approved*, we can confirm that staging is all set and push it to production.

*Assuming the current web issues around S3 are resolved. Heroku is having a hard time with those right now

**Comment by alanmoo** [comment]
(@mmmavis I just assigned #515 to you for review because if there are any other changes they should just be a CSS color string.)

**Comment by hannahkane** [comment]
@alanmoo - I know you're probably gone for the day. Can we plan to circle up early PST tomorrow and figure out our deployment plan? cc @edrushka

**Comment by alanmoo** [comment]
Closing this as everything is live except the tweet, which has its own ticket.

**Comment by edrushka** [comment]
🎺 🎉 🎺 🎉  Thanks everyone!

**Comment by hannahkane** [comment]
Yay! This process felt good. Thanks @edrushka @alanmoo @sabrinang and @mmmavis!!!
